### ISSUE TITLE
ScanNet, Only Using a Partition of the Data

### ISSUE THREAD

**Issue by Bozcomlekci** [drprojects/superpoint_transformer#158]
Greetings, I'm Batuhan, 

I'm only trying to use the subset of the dataset ScanNet for validation but getting this message:

```
The dataset must be organized into the following structure:
superpoint_transformer/data/scannet/
        └── raw/
            ├── scannetv2-labels.combined.tsv
            ├── scans/
            │   └── {scan_name}/
            │       ├── {scan_name}.aggregation.json
            │       ├── {scan_name}.txt
            │       ├── {scan_name}_vh_clean_2.0.010000.segs.json
            │       └── {scan_name}_vh_clean_2.ply
            └── scans_test/
                └── {scan_name}/
                    └── {scan_name}_vh_clean_2.ply
```

I suppose the dataloader expects the full dataset to be available. Is there any way to bypass this message in the validation step?

Thanks in advance.

**Comment by drprojects** [comment]
Hello @Bozcomlekci, thanks for your interest in the project. As you understood, the whole dataset is expected. If only you want to play around with a smaller version of the dataset for experimentation purposes, we allow you to do so by setting `datamodule.mini=True'. 

If (from what I understand) what you want is to not download the files for the train set but only specific ones from the validation or test set, this is not something we officially support. Still, you should be able to do so yourself without too much trouble by having a look at the dataset documentation and code. In your specific case, I suggest you have a look at how `MiniScanNet` works and create a similar class that only requires the files you are interested in.

### NB: If you ❤️ or simply use this project, don't forget to give the repository a ⭐, it means a lot to us !

**Comment by narges-tk** [comment]
> Hello @Bozcomlekci, thanks for your interest in the project. As you understood, the whole dataset is expected. If only you want to play around with a smaller version of the dataset for experimentation purposes, we allow you to do so by setting `datamodule.mini=True`.
> 
> If (from what I understand) what you want is to not download the files for the train set but only specific ones from the validation or test set, this is not something we officially support. Still, you should be able to do so yourself without too much trouble by having a look at the dataset documentation and code. In your specific case, I suggest you have a look at how `MiniScanNet` works and create a similar class that only requires the files you are interested in.
> 
> ### NB: If you ❤️ or simply use this project, don't forget to give the repository a ⭐, it means a lot to us !

Hi!
Can you please let me know where should datamodule.mini=True be set? Actually, I am getting an error for preprocessing small files (under 600 points), but for larger files from the same data, it's ok. Can it resolve my issue as well?
